src/unique-entities-are-same.ts

```diff
@@ -0,0 +1,12 @@
+import { EntityAttrs } from './types';
+
+export function uniqueEntitiesAreSame(
```

Comment:
The name is confusing. What does this have to do with unique entities? The parameters are two normal entityAttrs... Couldn't the name be simplified to sth like `isSameArray`.
[In case that's not possible: pls add a comment why that is)